consul_config.py.ctmpl

```diff
@@ -0,0 +1,51 @@
+{{- define "KEY" -}}
+    {{ key (printf "docker-server-configs/AB/config.%s.json/%s" (env "DEPLOY_ENV") .) }}
+{{- end -}}
+
+{{- define "KEY_ARRAY" -}}
+    {{- range $index, $element := (key (printf "docker-server-configs/AB/config.%s.json/%s" (env "DEPLOY_ENV") .) | parseJSON) -}}
+        "{{.}}",
+    {{- end -}}
+{{- end -}}
+
+import os
+
+DEBUG = False
+SECRET_KEY = '''{{template "KEY" "secret_key"}}'''
+
+{{if service "acousticbrainz-pgbouncer"}}
+{{with index (service "pgbouncer-master") 0}}
+SQLALCHEMY_DATABASE_URI = "postgresql://acousticbrainz:acousticbrainz@{{.Address}}:{{.Port}}/acousticbrainz"
+POSTGRES_ADMIN_URI="postgresql://postgres@{{.Address}}:{{.Port}}/template1"
+POSTGRES_ADMIN_AB_URI="postgresql://postgres@{{.Address}}:{{.Port}}/acousticbrainz"
+{{end}}
+{{end}}
+
+{{if service "listenbrainz-redis"}}
```

Comment:
should probably consider renaming this then :P

Reply:
Ah, yes. Well, it used to be standalone until we moved to production. We should've renamed it then...